### PR TITLE
A11y changes

### PR DESCRIPTION
**Pull request by pandeymauli**
_Description of what this PR is changing or adding, and why:_
- Fixed styling issues in codelab.md and accessibility.md
- Update the video link for app preview on TalkBack in codelab.md 

_Issues fixed by this PR (if any):_ Fixes #ISSUE-NUMBER

## Presubmit checklist
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/get-started/codelab.md
Comment (khanhnwin) [inline src/get-started/codelab.md:786]: this should probably use {{site.youtube-site}}

File: src/get-started/codelab.md
Comment (khanhnwin) [inline src/get-started/codelab.md:785]: should this comment be removed?

**Comment by khanhnwin** [comment]
Left a few comments, but will wait for @sfshaza2 to land. Thanks @pandeymauli!

File: src/get-started/codelab.md
Comment (atsansone) [inline src/get-started/codelab.md:401]: **issue:** With two part sentences, write as _why_ then _how_.
```suggestion
    To learn how to find and customize Android's accessibility features, view this video.

```

File: src/get-started/codelab.md
Comment (atsansone) [inline src/get-started/codelab.md:415]: **issue:** With two part sentences, write as _why_ then _how_.
```suggestion
    To learn how to find and customize iOS accessibility features, view this video.

```

File: src/get-started/codelab.md
Comment (pandeymauli) [reply src/get-started/codelab.md:358]: People tended to miss the second sentence if it was merged into the first box. That is why we separated it into two boxes.

File: src/get-started/codelab.md
Comment (pandeymauli) [reply src/get-started/codelab.md:87]: @Sfshaza and @khanhnwin, can you make this change? it requires updating the entire list, which I did not modify or test in my work.

File: src/development/accessibility-and-localization/accessibility.md
Comment (pandeymauli) [reply src/development/accessibility-and-localization/accessibility.md:156]: Done!

File: src/development/accessibility-and-localization/accessibility.md
Comment (pandeymauli) [reply src/development/accessibility-and-localization/accessibility.md:151]: Accessibility.md lists the recommendations above this section.

**Comment by pandeymauli** [comment]
I have made the changes requested by @atsansone and pushed them to my fork. Should I close this PR and create a new one?

File: src/get-started/codelab.md
Comment (khanhnwin) [reply src/get-started/codelab.md:85]: Thanks for looping me in! Let's shelve this for now. Since it's independent of the accessibility content, we should review with @sfshaza2 separately and open a different PR to update the list structure.

File: src/get-started/codelab.md
Comment (atsansone) [reply src/get-started/codelab.md:356]: Stet for now.

**Review by atsansone — APPROVED** [review]

**Review by khanhnwin — APPROVED** [review]

**Comment by khanhnwin** [comment]
Thanks @pandeymauli! @sfshaza2 can you PTAL?

**Review by sfshaza2 — APPROVED** [review]
lgtm. I see a few minor formatting issues, but nothing worth holding this PR up.

**Comment by sfshaza2** [comment]
Thanks, @pandeymauli!

**Pull request merged**
